checkov/terraform/graph_builder/graph_components/attribute_names.py

```diff
@@ -4,10 +4,9 @@
 
 
 @dataclass
-class CustomAttributes(CustomAttributes):
-    SOURCE_MODULE = "source_module_"
+class CustomTerraformAttributes(CustomAttributes):
```

Comment:
@gruebel Can we avoid this renaming?

Reply:
let me try it again, I had some issues with it, that's why I changed it.

Reply:
😄 I found a way, please have a look into my new commit